Procfile

```diff
@@ -0,0 +1 @@
+web: java -jar duckling.jar -p 80 -d .
```

Comment:
I'm not positive, but I think the standard in Heroku is to use a non-80 port for your app server.  And then there's something about configuring that port in the dashboard? or maybe you just need to pull the port from the environment.